### PR TITLE
When you run create contextualization with a new optional tx

### PR DESCRIPTION
**Pull request by ponyjackal**
fix OU-1193
fix OU-1194

### CODE REVIEW

File: src/commands/createContextualizer.ts [hunk 1]
```diff
@@ -53,9 +56,21 @@ export function registerCreateContextualizerCommand() {
...
+        // fetch transaction if hash is provided
+        if (options.hash) {
+          const txHashShorten = shortenTxHash(options.hash);
+          // grab transaction from api and save it in test/transactions
+          await grabTx(options.hash, name);
+          // add test transaction file name to template data
+          data['txHashShorten'] = txHashShorten;
+        } else {
+          // TODO; set default shorten hash
+          data['txHashShorten'] = '0x20f3ee91';
```
Comment (pcowgill) [inline src/commands/createContextualizer.ts:70]: @ponyjackal Let's change this before merging

**Review by pcowgill — CHANGES_REQUESTED** [review]
Once we change the default shortened txHash, this is good to go.

File: src/commands/createContextualizer.ts (hunk 1 above)
Comment (ponyjackal) [reply src/commands/createContextualizer.ts:70]: Do you have any preferred one as a default?

Comment (pcowgill) [reply src/commands/createContextualizer.ts:70]: Something more like `<INSERT_TX_HASH_HERE>` maybe?

Comment (ponyjackal) [reply src/commands/createContextualizer.ts:70]: I prefer `<INSERT_TX_HASH_HERE>`  ))

**Review by pcowgill — APPROVED** [review]

**Pull request merged**